app/mailers/rent_mailer.rb

```diff
@@ -4,6 +4,7 @@ class RentMailer < ApplicationMailer
   def rent_period_end(rent_id)
     @rent = Rent.find(rent_id)
     @user = User.find(@rent.user_id)
-    mail to: @user.email, subject: 'Your rent expired today'
+    I18n.locale = @user.try(:locale) || I18n.default_locale
```

Comment:
You will probably have this lines in several places. I recommend you to set the I18n locale in the api controller